### PR TITLE
Fix README - npm install fails (name mismatch)

### PR DESCRIPTION
**Pull request by theSKa**
``` bash
npm ERR! Invalid Package: expected styleguide-components but found tw-styleguide-components
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Live demo http://transferwise.github.io/styleguide-components/
 It is recommended to use bower to import the latest version
 ```
 dependencies: {
-    "styleguide-components": "https://github.com/transferwise/styleguide-components.git#v1.2.0",
+    "tw-styleguide-components": "https://github.com/transferwise/styleguide-components/archive/1.4.0.tar.gz"
```
Comment (StevePole) [inline README.md]: Why switch to archive over just updating the version?

Comment (theSKa) [inline README.md]: Hmm, no. 
screen shot 2016-10-04 at 19 20 58

Comment (theSKa) [inline README.md]:
```json
{
  "name": "steve",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
    "dependencies": {
      "styleguide-components": "https://github.com/transferwise/styleguide-components.git#v1.4.0"
    }
}
```

Comment (theSKa) [inline README.md]: And the name i changed because:
screen shot 2016-10-04 at 19 23 31

**Pull request closed without merging**